xbmc/settings/dialogs/GUIDialogContentSettings.cpp

```diff
@@ -336,6 +328,18 @@ void CGUIDialogContentSettings::InitializeSettings()
     return;
   }
 
+  AddButton(group, SETTING_CONTENT_TYPE, 20344, 0);
+  AddButton(group, SETTING_SCRAPER_LIST, 38025, 0);
+  CSettingAction *subsetting = AddButton(group, SETTING_SCRAPER_SETTINGS, 10004, 0);
+  if (subsetting != NULL)
+    subsetting->SetParent(SETTING_SCRAPER_LIST);
+
+  group = AddGroup(category, 20322);
+  if (category == NULL)
```

Comment:
I think this is null checking the wrong thing, probably should be group == null.

(Coverity 1355448)